source/_data/themes/ruyun.yml

```diff
@@ -0,0 +1,14 @@
+description: An ICP-friendly personal blog theme for Hexo
+link: https://github.com/ztygalaxy/hexo-theme-ruyun
+preview: https://tyzhang.top
+tags:
+  - hexo-theme
+  - Ruyun
+  - Customizable
+  - ICP
```

Comment:
Also, this one. What is `ICP`?

Reply:
`ICP` stands for `Internet Content Provider`, which is a licensing system that allows businesses to legally operate online services in China. This theme could config it friendly.

Reply:
I looked into the ICP license, but it seems that the ICP license is granted (registered) for websites, not for themes, correct?

Reply:
Yes,in China, ICP needs to be forced to be displayed at the bottom of the website. This theme will read from the configuration and display it at the bottom as required.